src/transcribo_backend/routes/transcribe_route.py

```
@@ -0,0 +1,100 @@
+
+from transcribo_backend.container import Container
+from transcribo_backend.helpers.file_type import is_audio_file, is_video_file
+from transcribo_backend.models.task_status import TaskStatus
+from transcribo_backend.models.transcription_response import TranscriptionResponse
+from transcribo_backend.services.whisper_service import WhisperService
+
+logger = get_logger(__name__)
+
+
+@inject
+def create_router(  # noqa: C901
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `C901`)

Remove unused `noqa` directive

(RUF100)